app/uk/gov/hmrc/agentsubscriptionfrontend/GuiceModule.scala

```diff
@@ -113,6 +113,7 @@ class GuiceModule(environment: Environment, configuration: Configuration) extend
     bindConfigProperty[String]("sosRedirectUrl")
     bindConfigProperty[Int]("mongodb.knownfactsresult.ttl")
     bindConfigProperty[Boolean]("agentAssuranceFlag")
+    bindConfigProperty[String]("r2dw-agentsAssurance-key")
```

Comment:
what is the reason to have multiple r2dw lists?

Reply:
There is just one collection, with multiple Properties(key, value), where the key represents different list.
so r2dw-agentsAssurance-key: r2dw list
and there will be a different key for MAA list

Reply:
but that information should be implicit, as I understand we've got 2 set of endpoints, one for r2dw and other for manually assured?